test/test_retriever.py

```diff
@@ -681,13 +688,22 @@ def test_reset_retriever(tmpdir):
     pwd_name = os.getcwd()
     workdir = tmpdir.mkdtemp()
     workdir.chdir()
-    dataset = random.choice(rt.dataset_names())
+    offline_datasets = rt.dataset_names()['offline']
+    test_datasets = ['test-eco-level-four', 'test-raster-bio1', 'test-raster-bio2',
+     'test-raster-multi-bands', 'test-raster-single-band', 'test-us-eco', 'test-vector']
+    offline_datasets = [dataset for dataset in offline_datasets if dataset not in test_datasets]
+    if len(offline_datasets) == 0:
+        return
+    dataset = random.choice(offline_datasets)
     rt.reset_retriever(dataset)
     rt.reload_scripts()
-    assert dataset not in rt.dataset_names()
-    rt.check_for_updates()
+    assert dataset not in rt.dataset_names()['offline']
```

Comment:
Looks like we can make two test out of this, online and offline. In most cases if the test code goes past 10 line, try to refactor it, unless otherwise